docs/technical/execution_controller.md

```diff
@@ -1,147 +1,149 @@
-    - (1B) status is up-to-date and phase = Failed
-    - (1C) status is not up-to-date or phase is not completed (in work)
+The error handling is similar to that of the installation controller, i.e. we distinguish normal and fatal errors. 
+Normal errors lead to a retry of the current phase. Fatal errors change the phase to `Failed` (resp. `DeleteFailed`), 
+so that the current flow is finished.
 
-- Second, the class of all pairs whose deploy item resource does not exist or whose spec is not up-to-date.
-  These deploy items need to be updated by the execution controller. We devide them in the following sub-classes:
+#### Starting Another Reconcile Flow
 
-    - (2A) no pending dependencies
-    - (2B) pending dependencies
+A new reconcile job can only be started at the root installation, and only when the root installation
+(and thus the complete object tree) is finished.
+During such a job, the execution spec can be updated by the controller of its parent installation. 
+Such a spec update alone does not trigger a reconcile of the execution.
+The execution controller waits until the execution has received the new job ID. Then the execution phase is changed to
+`Init` and the new reconcile flow of the execution starts.
 
-In case of a force-reconcile, we consider all pairs as belonging to the second class, i.e. we treat them as if the
-deploy item would need an update.
+Note that a new job ID triggers a new reconcile regardless whether or not the execution spec was modified.
 
-Based on this classification, the reconcile logic is as follows:
-
-- If all deploy items are in class (1A) (no update required and succeeded),
-  then the controller collects the export data, sets the execution phase = Succeeded, and returns.
+#### Deleting an Execution
 
-- If there exists a deploy item in class (1B) (no update required and failed),
-  then the controller sets the execution phase = Failed and returns.
+The deletion of an execution is always part of a job that has started with the deletion of the root installation and 
+propagates through the object tree. During the job, the execution gets a deletion timestamp in
+[phase "InitDelete" of the parent installation](installation_controller.md#phase-initdelete). 
+When it also has received the job ID in 
+[phase "TriggerDelete" of the parent installation](installation_controller.md#phase-triggerdelete)), 
+the deletion flow of the execution starts with phase `InitDelete`.
 
-- Otherwise, perform an update for all deploy items in class (2A) (update required and no pending dependencies).
-  Update also the [generations in the execution status](#generations-spec-up-to-date). 
-  Set the execution phase = Progressing and return.
+#### Phase "InitDelete"
 
+In phase `InitDelete`, the controller deletes all deploy items that are controlled by the execution.
 
-### Re-Check of Deploy items
+Note the modified behavior of this phase if the installation has the [interrupt annotation](#interrupting-a-job).
```

Comment:
I could not find the description of this behaviour.

Reply:
Done